standalone/src/DelphesPythia8EvtGenReader_k4Interface.h

```diff
@@ -122,12 +122,18 @@ class DelphesPythia8EvtGenReader_k4Interface: public DelphesInputReader {
     // new interface:
     //Pythia8::Pythia *pythia=m_pythia.get();
 
-    int regenerate=atoi(argv[10]);
+    int regenerate=0;//=atoi(argv[10]);
```

Comment:
What happends to `argv[10]`? It will just be ignored?

Reply:
It will not be ignored. The default value is 0. Now if people pass 11 arguments only then the argv[10] is used. This happens couple of lines below:
      {
      	m_evtgen2->add_decays(argv[7], atoi(argv[8]), argv[9]);
	      regenerate=atoi(argv[10]);
      }
If people pass 7 arguments then the configuration is for the inclusive sample and then we do not do any regeneration   by definition.